public/src/services/post_service.ts

```diff
@@ -75,6 +76,30 @@ export class PostService {
             }
         }
     }
+    
+    async renderPost(post: Post, pid: string, feedPost: boolean): Promise<string> {
```

Comment:
mind moving this to a `src/renderers/post_renderer.ts`?

Reply:
Done!